bigtable/CMakeLists.txt

```diff
@@ -195,7 +197,7 @@ set(bigtable_admin_unit_tests
     admin/admin_client_test.cc
     admin/column_family_test.cc
     admin/table_admin_test.cc
-    admin/table_config_test.cc)
+    admin/table_config_test.cc client/data_client.cc client/data_client.h)
```

Comment:
Why are these files on the same line, vs. different lines as all the other files/

Reply:
Also, why is the test depending on header/implementation files directly, shouldn't we have a library that contains the data client files, and tests should depend on that library? What about transitive dependencies?

Reply:
Fixed.  That was a mistake (or my IDE being too helpful).